mods/f1-blocker.wh.cpp

```diff
@@ -0,0 +1,28 @@
+// ==WindhawkMod==
+// @id              f1-blocker
+// @name            F1 Blocker
+// @description     Prevent F1 from opening the help page in File Explorer and/or other selected applications
+// @version         0.0.1
+// @author          d0gkiller87
+// @github          https://github.com/d0gkiller87
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*...*/
+// ==/WindhawkModReadme==
+
+BOOL( *TranslateAcceleratorW_Original )( HWND hWnd, HACCEL hAccTable, LPMSG lpMsg );
+BOOL TranslateAcceleratorW_Hook( HWND hWnd, HACCEL hAccTable, LPMSG lpMsg ) {
```

Comment:
The calling convention is missing here, which is likely to cause a crash for 32-bit targets. Please add calling conventions. See:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions